app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Typography, Dropdown, Tooltip, Avatar } from "@bigbinary/neetoui";
+import PropTypes from "prop-types";
+
+const Card = props => {
+  const { note, action } = props;
+  return (
```

Comment:
Always add a newline before the `return` statement. It denotes existing out of a scope.